src/mon/OSDMonitor.cc

```diff
@@ -4791,6 +4791,9 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
       rdata.append(ss.str());
       ss.str("");
     }
+  } else if (prefix == "osd get-require-min-compat-client") {
+    ss << ceph_release_name(osdmap.require_min_compat_client);
```

Comment:
use rdata.append(...)
ss goes to stderr, rdata goes to stdout.

Reply:
@hansbogert untagged untill @sage 's comments implemented

Reply:
@liewegas then shouldn't the `grep` in the test have failed, as that `grep` only checks stdout ?

Reply:
currently, yes.  we move a lot of the make check tests out to teuthology because they took too long to run.  we might consider putting the cli tests back in!